<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Receitas</string>
    </property>
    <widget class="QPushButton" name="btn_receitas_adicionar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_receitas_gerenciar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gerenciar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Ingredientes</string>
    </property>
    <widget class="QPushButton" name="btn_ingredientes_adicionar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ingredientes_gerenciar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gerenciar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fabricações</string>
    </property>
    <widget class="QPushButton" name="btn_fabricacoes_adicionar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fabricacoes_gerenciar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gerenciar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Vendas</string>
    </property>
    <widget class="QPushButton" name="btn_vendas_adicionar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_vendas_gerenciar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gerenciar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIngredientes">
    <property name="title">
     <string>Ingredientes</string>
    </property>
    <addaction name="action_ingredientes_adicionar"/>
    <addaction name="action_ingredientes_gerenciar"/>
   </widget>
   <widget class="QMenu" name="menuReceitas">
    <property name="title">
     <string>Receitas</string>
    </property>
    <addaction name="action_receitas_adicionar"/>
    <addaction name="action_receitas_gerenciar"/>
   </widget>
   <widget class="QMenu" name="menuFabricacoes">
    <property name="title">
     <string>Fabricações</string>
    </property>
    <addaction name="action_fabricacoes_adicionar"/>
    <addaction name="action_fabricacoes_gerenciar"/>
   </widget>
   <widget class="QMenu" name="menuVendas">
    <property name="title">
     <string>Vendas</string>
    </property>
    <addaction name="action_vendas_adicionar"/>
    <addaction name="action_vendas_gerenciar"/>
   </widget>
   <addaction name="menuIngredientes"/>
   <addaction name="menuReceitas"/>
   <addaction name="menuFabricacoes"/>
   <addaction name="menuVendas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_ingredientes_adicionar">
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="action_ingredientes_gerenciar">
   <property name="text">
    <string>Gerenciar</string>
   </property>
  </action>
  <action name="action_receitas_adicionar">
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="action_fabricacoes_adicionar">
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="action_vendas_adicionar">
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="action_receitas_gerenciar">
   <property name="text">
    <string>Gerenciar</string>
   </property>
  </action>
  <action name="action_fabricacoes_gerenciar">
   <property name="text">
    <string>Gerenciar</string>
   </property>
  </action>
  <action name="action_vendas_gerenciar">
   <property name="text">
    <string>Gerenciar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
